packages/babel-cli/src/babel/dir.ts

```diff
@@ -1,18 +1,16 @@
-// @flow
-
 import slash from "slash";
 import path from "path";
 import fs from "fs";
 
 import * as util from "./util";
-import { type CmdOptions } from "./options";
+import type { CmdOptions } from "./options";
 
 const FILE_TYPE = Object.freeze({
   NON_COMPILABLE: "NON_COMPILABLE",
   COMPILED: "COMPILED",
   IGNORED: "IGNORED",
   ERR_COMPILATION: "ERR_COMPILATION",
-});
+} as const);
```

Comment:
The return value of `Object.freeze` is read-only object so explicit `as const` is unncesary?
[Playground link](https://www.typescriptlang.org/play?#code/MYewdgzgLgBAYgSQDIFED6AVAmgBRTAXhgHkAjAKwFNgoA6AMwCdLKAvSgCgG8AoGGAHLEBaAMLEAsjmQBBAEKoAXDABEQkeKmyFKFQBo+MTdNQARZSuPIUp-YYQBxIQCUbFxy5t3+KZ87GSJjIYCMIWvv5WSMGhAnYAvgCUANw8PIiomLgotFE2hKpwxMQqqUA)

Reply:
without `const` values in object would be typed as `string`, but with it - as string literals

allowing type checks [like this](https://www.typescriptlang.org/play?#code/MYewdgzgLgBAYgSQDIFED6AVAmgBRTAXhgHkAjAKwFNgoA6AMwCdLKAvSgCgG8AoGGAHLEBaAMLEAsjmQBBAEKoAXDABEQkeKmyFKFQBo+MTdNQARZSuPIUp-YYQBxIQCUbFxy5t3+KZ87GSJjIYCMIWvv5WSMGhAnYAvjAAhhAwoJBQAJQA3Dw8UACeAA6U8ACWADaUGMWlRADWlAUg9DCFJS3w1pi4KLk8VbD0ldW1ynAjNSX9APQzMCD1PMNVU3VdqD14tFE2uXMwlIyMIIzLk7WEMADkKUkAJte5QA)

Reply:
Ah okay!